app/views/budgets/index.html.erb

```diff
@@ -83,10 +83,10 @@
       </div>
 
       <% unless current_budget.informing? %>
-        <div id="map">
+        <!-- <div id="map">
           <h3><%= t("budgets.index.map") %></h3>
           <%= render_map(nil, "budgets", false, nil, @budgets_coordinates) %>
```

Comment:
It's always a good idea to comment out ruby code, as html comments I think doesn't do that.